ansible/roles/common/templates/etc/pouta_blueprints/app_config.j2

```diff
@@ -1,6 +1,7 @@
 DEBUG: {{ application_debug_logging | bool }}
 SECRET_KEY: '{{ application_secret_key }}'
 SQLALCHEMY_DATABASE_URI: '{{ application_database_uri }}'
+MESSAGE_QUEUE_URI: 'redis://redis:6379/0'
```

Comment:
You mean configure redis to point to localhost in /etc/hosts ?
Because right now, in devel version the uri is:  redis://api:6379/0